showcase/public/index.js

```diff
@@ -1,6 +1,7 @@
-import { Elm } from './../src/Main.elm';
-import paackSvgIconSprite from 'paack-ui-assets/js/paackSvgIconSprite';
+import { Elm } from './../src/Main.elm'
```

Comment:
Probably this is by default, but you changed the code style here( removing the `;` everywhere) I think we should leave as it was (also change your file to contain `;`) and then in another pr/discussion we can decide in a JS code style and add the config file to the project, WDYT?